### DISCUSSION TITLE
Not working on Hololens2

### DISCUSSION THREAD

**Discussion by mattirck** [HISPlayer/Unity_Video_Player#10] [Q&A]
Hi,

I have problems to get the player working on Hololens2. Either it says 

DllNotFoundException: Unable to load DLL 'HISPlayerWindowsDLL'. Tried the load the following dynamic libraries: Unable to load dynamic library 'HISPlayerWindowsDLL' because of 'Failed to open the requested dynamic library (0x06000000) - The specified module could not be found.  (WinError:0000007e)

in the player.log or

DllNotFoundException: Unable to load DLL 'HISPlayerWindowsDLL'. Tried the load the following dynamic libraries: Unable to load dynamic library 'HISPlayerWindowsDLL' because of 'Failed to open the requested dynamic library (0x06000000) - %1 is not a valid Win32 application.  (WinError:000000c1)

For the first, there were the settings for the HISPlayerWindowsDLL:

![grafik](https://github.com/user-attachments/assets/35f15c7a-7436-4eb6-85b5-d8429ad14120)

Then I switched the settings to the following and now the second exception is occuring.

![grafik](https://github.com/user-attachments/assets/2f367a2a-fc26-4364-aca5-29e0eb77b50c)

The Unity-Version is 2022.3.24f1, Target Platform is UWP and it's the latest VideoPlayer-release from GitHub:

![grafik](https://github.com/user-attachments/assets/95c704a0-2426-45d6-94fb-93e12a88e44a)

**Comment by NurikHISPlayer** [comment]
Hi @mattirck ,

We will release our Unity SDK for Hololens in Q1 2025.

**Comment by mattirck** [comment]
So does this mean, that currently no Build for HL2 is possible?

**Reply by NurikHISPlayer** [reply]
Unfortunately, that is correct.

**Reply by mattirck** [reply]
What would be a sufficient demand?

**Reply by mattirck** [reply]
On https://hisplayer.com/unity-player-sdk/ you advertise Hololens as a Use Case? I thought, the implemenation of this feature is already finished?

**Reply by NurikHISPlayer** [reply]
We will release a new version.
